jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -213,7 +213,7 @@ def ensureEvalsDir() {
 def updateManifestVariable(manifestFileTxt, name, value) {
     if (name && value) {
         println("Updating manifest variable: name = ${name}, value = ${value}")
-        manifestFileTxt = manifestFileTxt.replaceFirst(/${name}: '.*'/, "${name}: '${value}'")
+        manifestFileTxt = manifestFileTxt.replaceFirst(/${name}: .*/, "${name}: '${value}'")
```

Comment:
What difference did this make?

Reply:
When it has the `'`, it doesn't catch the versions if the version isn't surrounded by it. Version of the [backup](https://github.com/integr8ly/installation/blob/backup-container-next/inventories/group_vars/all/manifest.yaml#L95) is an example of this. Removing the `'` should now update versions  surrounded with or without `'`.